package.json

```
@@ -18,7 +18,8 @@
     "lint:format": "prettier . --cache --check",
     "lint:js": "eslint . --cache",
     "lint:js:fix": "eslint . --fix",
-    "test": "vitest"
+    "test": "vitest",
+    "test:lib": "vitest lib/**/*.test.js"
```

It feel like `pnpm test` should run all tests.
I'm wondering whether the following would be good enough or 
```suggestion
    "test": "vitest **/*.test.js"
```
alternative in case the `**/*.test.js` glob ends up doing something funny
```suggestion
 "test": "vitest tests/**/*.test.js, lib/**/*.test.js"
```

---

The command `"test": "vitest"` means that `pnpm test` runs all the tests of all the project. vitest looks for `.test.js` extension by default.

The reason why I introduced `"test:lib": "vitest lib/**/*.test.js"` is because I want a command that runs specifically the tests in the `lib/` folder. 

**Detailed version**:

We have two types of tests in the project:
- Tests located in `tests/` are heavy tests that generate whole Ember apps to assert the compatibility of the codemod with the different Ember versions. 
- Tests located in `lib/` are light unit tests living in colocation with the file they test. We simply import the tested function and make sure it executes correctly.

`pnpm test` is not used on CI. Instead, we use a matrix and we run specifically `pnpm vitest -t ${{matrix.version}} tests/${{ matrix.file }}.test.js` to have results for each version.

`pnpm test:lib` is something that I need to use in a new CI job to run only the unit tests in `lib/`.

---

Gotcha :+1: I misunderstood then